engine/flink/management/dev-model/src/main/scala/pl/touk/nussknacker/engine/management/sample/service/MultiSelectEditorService.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.management.sample.service
+
+import io.circe.Json
+import pl.touk.nussknacker.engine.api.{MethodToInvoke, ParamName, Service}
+import pl.touk.nussknacker.engine.api.definition.MultiSelectFixedValue
+import pl.touk.nussknacker.engine.api.editor.{Editor, EditorType, LabeledExpression, MultiSelectLabeledValue}
+import pl.touk.nussknacker.engine.util.service.TimeMeasuringService
+
+import scala.concurrent.Future
+
+object MultiSelectEditorService extends Service with Serializable {
+
+  @MethodToInvoke
+  def invoke(
+      @ParamName("multiSelectParam")
+      @Editor(
+        `type` = EditorType.MULTI_SELECT_EDITOR,
+        possibleMultiSelectValues = Array(
+          new MultiSelectLabeledValue(value = "option1", label = "option1"),
+          new MultiSelectLabeledValue(value = "option2", label = "option2")
+        )
+      )
+      @Editor(`type` = EditorType.JSON_EDITOR)
+      multiSelect: Any,
```

Comment:
Correct me if I'm wrong, but I think `io.circe.Json` can be used here instead of `any`